drivers/aks/aks_driver.go

```diff
@@ -1,75 +1,140 @@
 package aks
 
 import (
-	"regexp"
-	"strings"
-
 	"context"
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
-	"time"
-
-	"github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2018-03-31/containerservice"
-	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2017-05-10/resources"
+	containerservice_20180331 "github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2018-03-31/containerservice"
```

Comment:
Add a space between the go imports and other imports.

Reply:
Fixed